src/components/ui/Footer.vue

```diff
@@ -1,43 +1,67 @@
 <template>
   <footer class="footer">
-    <a
-      v-for="(single, index) in links"
-      v-bind:key="index"
-      v-bind:href="single.link"
-      class="footer__link"
-      target="_blank"
-      rel="noopener noreferrer"
-    >
-      <Icon v-bind:id="single.icon" :width="12" :height="12" />
-      <abbr v-if="single.abbr" v-bind:title="single.abbr">
-        {{ single.label }}
-      </abbr>
-      {{ !single.abbr ? single.label : '' }}
-    </a>
-    <select v-if="languages.length > 1" v-on:change="updateLocale">
-      <option
-        v-for="[locale, language] in languages"
-        :key="locale"
-        :value="locale"
-        :selected="locale === currentLocale ? true : false"
-        >{{ language }}</option
+    <section class="footer__contribute" v-if="showContribute">
+      <h1>{{ fluent('contribute_header') }}</h1>
+      <div class="footer__contribute-img">
```

Comment:
I'm concerned with this class name as it doesn't seem follow BEM standards. Not saying that we HAVE to do that, but I'd like to work with that. If you think otherwise, that's fine, but I think it's worth discussion.

Reply:
Trying to fix all of those. Thanks. Still not doing this intuitively.